R/local.R

```diff
@@ -117,7 +117,18 @@ local_reproducible_output <- function(width = 80,
     .local_envir = .env,
   )
   withr::local_envvar(RSTUDIO = NA, .local_envir = .env)
-  withr::local_language(lang, .local_envir = .env)
+
+  if (isTRUE(capabilities("NLS"))) {
+    withr::local_language(lang, .local_envir = .env)
+  } else {
+    warning_msg <- paste0(
```

Comment:
I think you can just omit this — it's not going to affect much code, so in most cases it will be a false positive.

Reply:
Warning and related tests removed and an entry was added to `NEWS.md`.